desktop/src/com/mygdx/game/client/di/gameinstance/ComponentMessageListenerModule.java

```diff
@@ -53,6 +55,7 @@ public ComponentMessageListener provideComponentMessageListener(
     var listener = new ComponentMessageListener(mapper); // jobs
     listener.registerHandler(EntityConfigId.class, entityConfigHandler);
     listener.registerHandler(EntityConfigId.class, desktopEntityConfigHandler);
+    listener.registerHandler(PlayerMaterial.class, materialHandler);
```

Comment:
to usuń, przypadkowo nie usunąłem z konstruktora wcześniej

Reply:
teraz ten PlayerMaterialHandler ma kod, który obsługuje zapisywanie tych materiałów, tamten handler, który był bardziej desktopowy ma usunięty kod, który był wspólny

teraz jest ok